### ISSUE TITLE
Upgrade to fluent-bit 1.8.x

### ISSUE THREAD

**Issue by a-thaler** [kyma-project/kyma#10582]
**Description**
fluent-bit nowadays bundles the loki output plugin, so there is no need to bundle it manual. So let's simplify the setup by removing the custom loki plugin.

- [x] Upgrade fluent-bit ~~and remove custom build step for loki plugin~~
- [x] Adjust fluent-bit config so that it uses the built-in plugin and results in the same behaviour (check the label mapping, the config is quite different)
- [ ] Verify that everything works as before (logs arrive in grafana and kyma console) and the log stream set is the same

**Reasons**

**Attachments**

**Comment by a-thaler** [comment]
I tried to use the new plugin coming from fluent-bit. In general it works fine but I had a crucial problem. The labelMap needs to be specified in a different way using the Record Accessor Pattern of fluent-bit. That is doable and fine, but if one of that expression cannot be fullfiled, the whole label set will not be attached to the record and loki cannot even assign them to a stream uniquely.
For example, as one of the rare labels kyma is indexing out-of-the-box is the "app" label. That label might not exist for some pods. With that the record accessor is not valid for a specific log record, so that no label gets assigned at all. With that the log gets only the stream label assigned and Loki will most probably have a log for such stream and timestam already and will return a 400.

Because of that problem I switched to the strategy to use the old grafana plugin which is still supported but renamed to "grafana-loki"

**Comment by a-thaler** [comment]
Furthermore, the patch we applied to fix https://github.com/fluent/fluent-bit/issues/2661 is not compatible anymore and it turned out that the problem still persists in the newest version. So we need to postpone the upgrade till the problem is solved. 
#issuecomment-783950281

**Comment by a-thaler** [comment]
The problem with the loki output plugin persists and we need to solve it in a different issue as it is most important to finally upgrade to the newest fluent-bit version.
I made several tests with the newest image and it looks like the problems are gone. I had a cluster with an old fluent-bit version where the output silently stopped working every 10hours. Deploying the new version made the daemonset survive for 3 days already.

So I updated to the latest version, removed the old patch logic and update the custom sequentialhttp plugin
